resources/src_vendor/palo/palo_config_coverter.py

```diff
@@ -148,6 +155,14 @@ def address(self):
 
 
             if self.vendor == 'forti':
+                try:
+                    ip_network(address_ip, strict=True)
+                    # address_ip is a valid network address (or a valid host with /32 mask) -->nothing to do
+                    address_ip = address_ip.split("/")[0] +" "+str(ip_network(address_ip).netmask)
+                except ValueError:
+                    # address_ip is a palo specific host address that also has a network mask
+                    # --> convert to Host
+                    address_ip = address_ip.split("/")[0]+" 255.255.255.255"
```

Comment:
please share the palo address format

Reply:
In Palo alto you have can have 3 kinda different formats.
1. just the IP of an Host. for example "192.168.1.123"
2. The Network adress with a subnet mask in cidr notation. For example "192.168.1.0/24"
3. The IP address of an host with a subnet mask indicating in wich subnet the IP is used. As far as i am aware this is just for readability, and wont get used. for example "192.168.1.123/24"

Case 3 is what gets handled here ( in the except ValueError case).

This was the "cleanest" way i found to convert it into a simple host address.